src/sugar3/graphics/texteditor.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2007, Red Hat, Inc.
```

Comment:
It is an interesting choice to grant copyright of your work to Red Hat?  Have you been keeping it secret from when you used to work there in 2007 :smile: